src/index.ts

```diff
@@ -316,10 +327,14 @@ async function update() {
   tryUpdate();
 }
 
-async function getLatestVersionLink() {
+async function getLatestVersionName() {
   const LATEST_VERSION_URL = "https://api.github.com/repos/Rookout/explorook/releases/latest";
   const response = await fetch(LATEST_VERSION_URL);
-  const verNum = (await response.json()).name;
+  return (await response.json()).name;
```

Comment:
consider wrapping it in a try catch block the fetch can fail

Reply:
also consider checking that `resopnse.ok` is true and if not throw an error (you will catch it using the try-catch)

Reply:
// something like:
async function getLatestVersionName() {
  try {
    const LATEST_VERSION_URL =
      'https://api.github.com/repos/Rookout/explorook/releases/latest';
    const response = await fetch(LATEST_VERSION_URL);
    if (!response.ok) {
      throw new Error('Problem fetching the latest version')
    }
    return (await response.json()).name;
  } catch (e) {
    //handle error
  }
}

Reply:
@EliRookout This function is caught everywhere it is used.
line 295
line 323

Reply:
> @EliRookout This function is caught everywhere it is used. line 295 line 323

cool but you don't raise a new exception in case the `response.ok` is `false`

Reply:
> > @EliRookout This function is caught everywhere it is used. line 295 line 323
> 
> cool but you don't raise a new exception in case the `response.ok` is `false`

Done